components/Image/image.html

```diff
@@ -0,0 +1,113 @@
+                <li class="ma-item"><a class="ma-link" href="../Modal/modal.html">Modal </a></li>
+                <li class="ma-item"><a class="ma-link" href="../Rating/rating.html">Rating </a></li>
+                <li class="ma-item"><a class="ma-link" href="../Toast/toast.html">Toast </a></li>
+                <li class="ma-item"><a class="ma-link" href="../Grid/grid.html">Simplified Grid </a></li>
+            </ul>
+        </aside>
+
+
+    <!-- main body area -->
+        <main class="main">
+            <section class="intro">
+                <h1 class="para-heading" >Image</h1>
+                <p class="doc-para">
+                    To start using the image components in your project, use the code snipppets below.
+                </p>
+                             
+
+                <!-- ROUND IMAGE -->
+                <h2 class="para-heading" id="r-img">Round-Image</h2>
+                <p class="doc-para">
+                    A round image is the one that is circular in shape. Use class "img-round" for this component.
+
+                </p>
+                <div class="comp-container ">
+                    <img class="img-round" src="https://picsum.photos/200/300" alt="">
+                </div>
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28200%2C178%2C200%2C1%29&t=seti&wt=none&l=htmlmixed&width=746.25&ds=true&dsyoff=15px&dsblur=68px&wc=true&wa=true&pv=32px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522img-round%2522%2520src%253D%2522%2522%2520alt%253D%2522%2522%253E"
+                style="width: 100%; height: 157px; border:0; transform: scale(1); overflow:hidden; margin-bottom: 3rem; margin-top: 1rem;"
+                sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+                <!-- RESPONSIVE IMAGE -->
+                <h2 class="para-heading" id="res-img">Responsive-Image</h2>
+                <p class="doc-para">
+                    A Responsive image is the one that adjusts its height and width according to the screen size. Use class "img-responsive" for this component.
+                </p>
+                <div class="comp-container comp-img-container">
+                    <img class="img-responsive" src="https://picsum.photos/200/200" alt="">
+                </div>
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28200%2C178%2C200%2C1%29&t=seti&wt=none&l=htmlmixed&width=746.25&ds=true&dsyoff=15px&dsblur=68px&wc=true&wa=true&pv=32px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522img-responsive%2522%2520src%253D%2522%2522%2520alt%253D%2522%2522%253E"
+                style="width: 100%; height: 157px; border:0; transform: scale(1); overflow:hidden; margin-bottom: 3rem; margin-top: 1rem;"
+                sandbox="allow-scripts allow-same-origin">
+                </iframe>
```

Comment:
Suggestion: you can define a class for iframe and define corresponding stylings there and can reuse that class and that helps in responsiveness as well